src/protocol/protocol.ts

```diff
@@ -136,54 +152,31 @@ export namespace CommonDataTypes {
     sharedId: string;
   };
 
-  // UndefinedValue = {
```

Comment:
I actually like those comments because they make it clear what spec text was used to author types (not blocking opinion).

Reply:
+1 for this comment.

Reply:
OK, I'll revert the comment removals for now for the sake of being less contentious.

That said, I think it's not worth the effort to add these comments, plus they can get outdated easily as the spec evolves (e.g. whenever new subfields/keys are added to compound objects).

Reply:
All comments were re-added.